apache/superset
# feat: Update ShortKey for stop query running in SqlLab editor

### SUMMARY

- Change the shortcut for Stop Query from Control + X to Control + E on PC / Linux
- Update the tooltip text on the stop query button to reflect the shortcut change
- Mac shortcut should remain the same as it is today (Control+X)

### BEFORE/AFTER SCREENSHOTS OR ANIMATED GIF


### TESTING INSTRUCTIONS


1. Go to SqlLab
2. Run any query
3. Try to press ctrl + e(on PC, Linux) or ctrl + x (on Mac)
4. Check tool tip when hovering on `Stop query` button

### ADDITIONAL INFORMATION


- [x] Has associated issue: https://github.com/apache/superset/issues/9990
